style.css

```diff
@@ -36,4 +36,7 @@ h1
 }
 .myButton:active {
     background: #144E75; 
+}
+.link-container {
+    align-items: center;
```

Comment:
Think `align-items` only works if you have `display: flex` as well?

https://www.w3schools.com/CSSref/css3_pr_align-items.asp

Reply:
i'll have to play around with containers later, at least i am assuming i need to group the buttons in a container to move them around?

Reply:
Yea whatever you do to the container will move the 3 links inside. Easy to stack buttons horizontally, vertically, and adjust spacing. Nothing inherently special about a 'container' is just a name for an empty div element that has other elements as children.